packages/block-library/src/post-excerpt/index.php

```diff
@@ -27,7 +27,15 @@ function render_block_core_post_excerpt( $attributes, $content, $block ) {
 	* wp_trim_words is used instead.
 	*/
 	$excerpt_length = $attributes['excerptLength'];
-	$excerpt        = get_the_excerpt( $block->context['postId'] );
```

Comment:
The function already has a filter - `get_the_excerpt`.

Reply:
Hey @Mamaduka, so in one of the issues https://github.com/WordPress/gutenberg/issues/28214 it has been discussed already that, even though `get_the_excerpt()` function already uses the `get_the_excerpt` filter, It would be good to have `the_excerpt` filter for backwards compatibility.

Reply:
I got it. The issue concerns the PHP filter. There's no need to update the JS code. For various reasons, the blocks don't mirror PHP filters in their React components.

P.S. I would recommend using reserved [GitHub keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) to reference issues. It makes clear what you're trying to solve.